client/js/components/pages/HomePage.js

```diff
@@ -1,348 +1,32 @@
 import React, { Component } from 'react';
+import getGoogleApiClient from 'google-client-api';
 
-class HomePage extends Component {
-  constructor() {
-    super();
-    this.state = { visibility: false };
-  }
+import Logo from '../../../img/xteam-vertical.png';
+import 'file?name=[name].[ext]!../../../img/xteam-vertical.png';
 
-  hidePanel() {
-    this.setState({ visibility: true });
+class HomePage extends Component {
+  componentDidMount() {
+    getGoogleApiClient(gapi => {
+      gapi.load('signin2', () => {
+        gapi.signin2.render('g-signin2', {
+          longtitle: true,
+          width: 250,
+          height: 50
+        });
+      });
+    });
   }
 
   render() {
-    const visibility = this.state.visibility ? 'hidden' : '';
     return (
```

Comment:
It would mean that all URLs that don't match any of the routes defined in the routing would show the default view - there won't be a redirect or any information that URL is invalid. I'd rather just create empty homepage component - this way we can show empty map on "/", but show 404 or redirect to homepage on all invalid routes.